src/shogun/labels/MulticlassLabels.cpp

```diff
@@ -71,6 +71,19 @@ void CMulticlassLabels::allocate_confidences_for(int32_t n_classes)
 	m_multiclass_confidences = SGMatrix<float64_t>(n_classes,n_labels);
 }
 
+SGVector<float64_t> CMulticlassLabels::get_confidences_for_class(int32_t i)
+{
+	REQUIRE((m_multiclass_confidences.num_rows != 0) && 
+		(m_multiclass_confidences.num_cols != 0),
+		"Call allocate_confidences_for() before fetching confidences");
```

Comment:
missing newline

Reply:
I woudl also suggest to change the error msg
"Empty confidences, which need to be allocated before fetching.\n")

Reply:
Okay, that looks much better :)